controllers/index.js

```diff
@@ -1,9 +1,15 @@
 const router = require('express').Router();
 
 const apiRoutes = require('./api');
-const homeRoutes = require('./homeRoutes');
+const homeRoutes = require('./homeRoutes.js');
+const profileRoutes = require('./profileRoutes.js');
```

Comment:
I think this file is userRoutes and you don't need the extension

Reply:
Let's discuss tomorrow.

Reply:
I made this comment before I came across the profileRoutes file lower in the pull request.  I think this does require a userRotes in addition to the ones you have here.